eng/Versions.props

```diff
@@ -28,7 +28,7 @@
     <VisualStudioEditorPackagesVersion>17.5.143-preview</VisualStudioEditorPackagesVersion>
```

Comment:
> src/EditorFeatures/Core.Cocoa/Workspaces/CocoaTextBufferVisibilityTracker.cs(28,21): error CS1061: (NETCORE_ENGINEERING_TELEMETRY=Build) 'ICocoaTextView' does not contain a definition for 'IsVisible' and no accessible extension method 'IsVisible' accepting a first argument of type 'ICocoaTextView' could be found (are you missing a using directive or an assembly reference?)

Do we also need to bump the version?

Reply:
Oh weird, I must have been looking at a branch when I thought that `VisualStudioEditorPackagesVersion` had a _newer_ version now.

Sorry, this can wait until your team is ready to bump on Windows.